exec/jdbc-all/pom.xml

```diff
@@ -511,7 +511,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
 
                   </message>
-                  <maxsize>39000000</maxsize>
+                  <maxsize>39500000</maxsize>
```

Comment:
There was no need to increase the limit for Jar size since in https://github.com/apache/drill/pull/1486 were excluded some classes so the issue with hitting the limit without adding any library was fixed there.

Reply:
@vvysotskyi,
- I rebased several times and didn't see the fix; saw it only after publishing the PR
- Talked internally to the team and was advised to update the pom

BTW - The current way of packaging the jdbc-all jar is unsustainable (unless we accept the fact this jar might indefinitely increase in size). The right fix, is to divide the java-exec project into multiple projects: common, client, and server. This way, the jdbc-all artifact should only depend on common and client.